index.html

```diff
@@ -17,7 +17,7 @@
 
       /*set the background*/
       .bg {
-            background: url('https://jlprojects.com/wp-content/uploads/2017/06/Sea-Ranch-Boca-Raton-Condo-Remodeling-3.jpg');
+            background: url('index_img.jpg');
```

Comment:
Please put all images in a new folder named "img", just like all javascripts are in the folder "js" and css files are in "css".